test/SecretFileSpec.hs

```diff
@@ -3,21 +3,119 @@ module SecretFileSpec where
 import Test.Hspec
 
 import Data.Either (isRight, isLeft)
+import Control.Monad (forM_)
 
 import qualified SecretsFile
 import qualified System.Directory as Dir
 
 spec :: SpecWith ()
 spec = do
   describe "SecretFile.readSecretList" $ do
-    it "parses all golden tests succesfully" $ do
-      goldenTestContents <- Dir.listDirectory "test/golden"
-      let goldenTestFiles = map ("test/golden/" <>) goldenTestContents
-      parseResults <- mapM SecretsFile.readSecretsFile goldenTestFiles
-      parseResults `shouldSatisfy` (all isRight)
```

Comment:
I changed this to have one `it` per file, so that when the tests fail, it only prints the failing file, instead of a big list with and “`all isRight` is false for this big list, good luck finding the offending one!”.